### PR TITLE
upgraded kotlin version

### PR DESCRIPTION
**Pull request by fluidvanadium**
this seems necessary for integration tests

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlinVersion = '1.6.0'
+    ext.kotlinVersion = '1.7.10'
```
Comment (juanky201271) [inline android/build.gradle:4]: I remember we downgrade this version for some problem we had recently... @Oscar-Pepper do you remember this?

Comment (juanky201271) [reply android/build.gradle:4]: Let me try this version... maybe with some new other lib version could work...

Comment (juanky201271) [reply android/build.gradle:4]: I have no problem with Android Studio and the new kotlin version.

**Review by juanky201271 — APPROVED** [review]
I think this new kotlin version doesn't break anything...

**Comment by juanky201271** [comment]
@fluidvanadium I think you need to commit signed...

**Comment by zancas** [comment]
Superceded by my rebased version.

**Pull request closed without merging**